airflow/www/security.py

```diff
@@ -171,19 +171,25 @@ def __init__(self, appbuilder):
             view.datamodel = CustomSQLAInterface(view.datamodel.obj)
         self.perms = None
 
-    def init_role(self, role_name, perms):
```

Comment:
We possibly need to keep (but deprecate) this method -- as this is part of a quasi-public API and may be in use by people outside of the Airflow codebase.

Reply:
Done.